modules/ROOT/pages/keycloak/coding.adoc

```diff
@@ -9,74 +9,123 @@ This guide will help you to set up the {keycloak-service} service SDK in your Ap
 * xref:keycloak/provisioning.adoc[Provisioned {keycloak-service} service].
 * xref:keycloak/binding.adoc[Binding] created.
 * xref:keycloak/configuring-dev-env-keycloak.adoc[Configured dev environment].
-* xref:keycloak/keycloak-setup.adoc[Configured Keycloak].
+* Read the xref:configuring-dev-env.adoc[general app setup guide].
 
 == Importing the libraries
 
 [role="primary"]
 .Android
 ****
-. Import Core module.
-
-. Specify the schema of the redirect url for your Android project in `android.defaultConfig.manifestPlaceholders` of the apps `build.gradle` file.
+. Import the auth module
 +
-NOTE: It should match the schema specified in the `Valid Redirect URIs` section for the client in Keycloak.
-
-. TODO: Import auth module.
+[source,groovy,subs="attributes"]
+----
+dependencies {
+    implementation "org.aerogear:aerogear-auth:{release-number}"
+}
+----
 ****
 
 [role="secondary"]
 .iOS
 ****
-* Add dependency to your Podfile:
+. Add the dependency to your Podfile
++
+[source,ruby,subs="attributes"]
+----
+target '[TARGET NAME]' do
+    pod 'AGSAuth', '{release-number}'
+end
+----
+
+. Update the dependencies
 +
+[source,bash]
 ----
-pod 'AGSAuth'
+pod install
+----
+
+. Import the SDK
++
+[source,swift]
+----
+import AGSAuth
 ----
 ****
 
 [role="secondary"]
 .Cordova
 ****
+Install the auth package:
+[source,bash]
+----
+npm install @aerogear/auth
```

Comment:
Missing $

Reply:
I'm not sure. If people copy the whole line to paste into a terminal they dont want the $. But we should be consistent.

Reply:
@pb82 consistency is the most important, but also checked with other  writers, let's go with $
I know it's not ideal, but we can change later if required (easy to change if we're consistent)